Lib/fontmake/instantiator.py

```diff
@@ -517,9 +532,11 @@ def collect_info_masters(
 ) -> List[Tuple[Location, FontMathObject]]:
     """Return master Info objects wrapped by MathInfo."""
     locations_and_masters = []
+
     for source in designspace.sources:
         if source.layerName is not None:
```

Comment:
I think we want to get rid of this `if source.layerName is not None:` altogether, and only rely on the check `if source is not designspace.default:` in order to decide whether to consider this source's info or kerning or whether to skip it when it's not the default

Reply:
oh sorry sorry! I misunderstood.. this is gathering sources from all masters that contribute to info/kerning and we were previously skipping the 'sparse' sources.. let me think

Reply:
ok yes you're right, we need to check both layerName and whether source is not default. Please put the two if conditions on the same line using `and` operator

Reply:
I was just about to finish writing a comment to the same effect. Fixed, thanks!